### DISCUSSION TITLE
Coming Soon: Version 3 of Docker PHP Containers

### DISCUSSION THREAD

**Discussion by jaydrogers** [serversideup/docker-php#216] [Announcements]
Hey all,

Just wanted to give you an update on what to expect for **Version 3.0** of our containers. There's a lot to cover!

### What to expect in 3.0
We're working very hard to take these Docker images to the next level. That includes:
- Better Compatibility
- More Documentation
- Improved stability

### Video Update
Check out this video explaining what's in the works:
<a href="https://www.youtube.com/watch?v=JKR-zwwaJ1A"><img width="100%" alt="image" src="https://github.com/serversideup/docker-php/assets/3174134/343a1665-abe4-4832-8dd5-d936e51f1495"></a>

### Track our progress
The best place to stay updated is by watching this PR:
- https://github.com/serversideup/docker-php/pull/207

### Let us know your thoughts
After all, this is a **community** project -- so we're interested in hearing your thoughts! Any questions, let us know ✌️

**Comment by johnvo9595** [comment]
Please, add roadrunner!!!

**Reply by jaydrogers** [reply]
Here's a guide on how to request features. I will build them if there is a demand in the community for the feature: https://github.com/serversideup/docker-php/discussions/66
